.travis.yml

```diff
@@ -21,7 +21,7 @@ script:
   - npm --version
   - node_modules/.bin/builder run check-ci
   # Only run chromatic once
-  - if [ "${TRAVIS_NODE_VERSION}" = "6" ]; then npm run chromatic-test; fi
+  - if [ "${TRAVIS_NODE_VERSION}" = "6" ]; then npm run chromatic-test || 1; fi
```

Comment:
I'd perhaps suggest:

```yml
if [ "${TRAVIS_NODE_VERSION}" = "6" ]; then npm run chromatic-test || echo "Chromatic tests failed. Please investigate"; fi
```

or something to give a greppable message.

Reply:
Thanks, I'll make this change